packages/utilities/src/dom/getRenderedContainer.ts

```diff
@@ -0,0 +1,37 @@
+import { IPoint } from '../IPoint';
+import { Rectangle } from '../Rectangle';
+
+export function getRenderedContainer(
```

Comment:
Is there another name that would fit for this? Is `getContainerSurroundingContent` accurate? Without the context of `callout`, it's hard to discern how this should be used.
Alternately, if this is only going to be used by callout, could this just be a private method in the scope of callout?

Reply:
I wanted to keep `Rendered` in the name to make it clear that this refers to the rendered container, not the logical container.

Other things like dialog (and I think coachmark) will be using this function